Moise1/MyDiary
# #169447444 - Applying my LFA's feedback on Back-end

#### What does this PR do? 
- Returning a relevant message if no entries available yet (e.g: No entries yet).
- Protecting all entry-related routes for solely the entry owner.

#### Description of Task to be completed?

#### _Have the following working_ :
 - All the feedback provided by the LFA implemented and bugs fixed.
 
#### How should this be manually tested?(back-end)
- Clone this GitHub repo on your computer
- cd MyDiary
- install Postman if you don't have it.
- run  **npm install**  if you have node new version ready.
- run **npm start** 
- check all the mentioned changes regarding the back-end in the Postman.

#### Any background context you want to provide?
N/A
#### What are the relevant pivotal tracker stories?

[#169447444](https://www.pivotaltracker.com/n/projects/2402313)
